decidim-proposals/spec/system/filter_proposals_spec.rb

```diff
@@ -557,4 +557,61 @@
       end
     end
   end
+
+    context "when using the browser history", :slow do
+      let(:proposals) { create_list(:proposal, 2, component: component) }
+      let(:official_proposals) { create_list(:proposal, 2, :official, component: component) }
+      let(:accepted_proposals) { create_list(:proposal, 2, :official, :accepted, component: component) }
+      let(:rejected_proposals) { create_list(:proposal, 2, :official, :rejected, component: component) }
+
+      before do
+        proposals && official_proposals && accepted_proposals && rejected_proposals
```

Comment:
Instead of invoking them here you could declare them with `let!` to force the methods invocation before each example, and this line can be removed.

Reply:
mm, I was used to call the `let!` method, but after some rubocop warnings I got used to create things in the before block. I think that this is the cop I'm refering to: https://www.rubydoc.info/gems/rubocop-rspec/1.7.0/RuboCop/Cop/RSpec/LetSetup

The thing is that I'm doing it wrong anyway, because following that cop I should call the `create_list` methods in the before block instead of using a regular `let`. I understand that the idea of the cop is to avoid defining a `let` or `let!` that it will never be referenced.